Nguyen_Ken/Assignments/multiples.py

```diff
@@ -0,0 +1,10 @@
+#multiples assignment part 1
+for count in range(1,1001):
+    print count
+    count+=2
+
+
+#multiples assignment part 2
+for count in range(5, 1000001):
+    print count
+    count+=5
```

Comment:
okay i see it now, thanks for the link. will get that fixed  

    On Saturday, September 3, 2016 4:05 PM, authman <notifications@github.com> wrote:
 

 In Nguyen_Ken/Assignments/multiples.py:> @@ -0,0 +1,10 @@
> +#multiples assignment part 1
> +for count in range(1,1001):
> +    print count
> +    count+=2
> +
> +
> +#multiples assignment part 2
> +for count in range(5, 1000001):
> +    print count
> +    count+=5
doesn't work due to same reason commented above—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.